src/vscode.api.ts

```diff
@@ -0,0 +1,13 @@
+import {
```

Comment:
since there is no way to spy on `setDecorations` because it is a non-enumerable property, this wrapper was introduced.

Reply:
I don't like the idea of making such a change just for tests - Isn't there an api to get current decorations from the editor? If it exists, maybe you can use it in the tests to assert that our decoration exists?

Reply:
Well, I didn't like it either. obviously, the existence of such an API would make life much easier, but unfortunately, there is none (to remind you that we were together on a quest to find  one where you implemented the first iteration of the inline)